plugins/colorschemes/everforest.nix

```diff
@@ -0,0 +1,266 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+let
+  mkEverforestBool =
+    pluginDefault: desc:
+    helpers.defaultNullOpts.mkEnum [
+      0
+      1
+    ] pluginDefault desc;
```

Comment:
I think a type alias and/or option helper for `enum [0 1]` would be helpful in all of nixvim. Maybe in a future PR we'll do that